source/dmocks/mocks.d

```diff
@@ -772,6 +772,19 @@ unittest
     assert (calledPayload);
 }
 
+@("delegate payload with mismatching parameters")
+unittest
+{
+    Mocker r = new Mocker();
+    auto o = r.mock!(SimpleObject);
+
+    //o.print;
+    r.expect(o.print).action((int) { });
+    r.replay();
+
+    assertThrown!Error(o.print);
```

Comment:
We already use `@("...")` from unit-threaded, but we don't use unit-threaded for the unit test.
So, we cannot use, for example, the better `shouldThrowWithMessage`.